.github/workflows/redeliver.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: Redeliver failed webhooks
+
+# This workflow runs every 6 hours or when manually triggered.
+on:
+  schedule:
+    - cron: '20 */6 * * *'
+  workflow_dispatch:
+
+# This workflow will use the built in `GITHUB_TOKEN` to check out the repository contents. This grants `GITHUB_TOKEN` permission to do that.
+permissions:
+  contents: read
+
+jobs:
+  redeliver:
+  - name: Redeliver failed webhooks
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: gateixeira/retrigger-webhook-action@main
+      env:
+        token: ${{ secrets.GH_TOKEN_FOR_ACTIONS }}
+        owner: ${{ github.repository_owner }}
+        repo: ${{ github.event.repository.name }}
+        last_redelivery_variable_name: 'LAST_WEBHOOK_REDELIVERY'
```

Comment:
Where does that come from? I don't really understand the documentation of the action.

Reply:
Also, which webhook should it attempt to redeliver? we have more than one and probably need to give it an id, e.g. like documented `webhook_id: 'YOUR_HOOK_ID'`

Reply:
Ah. I forgot to add the comment here. This is to store the last timestamp in a variable.